### ISSUE TITLE
Pydantic v2.5.0 breaking installs due to size of Description

### ISSUE THREAD

**Issue by NodeJSmith** [pydantic/pydantic#8096]
### Initial Checks

- [X] I confirm that I'm using Pydantic V2

### Description

This is a weird one - nothing is wrong with Pydantic itself, but it seems that the package description including so much of the changelog is causing issues.

I am using an Azure Devops Artifact Feed, which pulls things through PyPi to mirror them, along with my company's own Python libraries. As of V2.5.0 I am getting an error installing Pydantic due to the size of the description. The error is below.

This seems to be due to a limitation that Azure DevOps artifact feed has on the Description, per [this issue](https://github.com/Azure/azure-cli/issues/18383) and [this issue on the VS Developer Community](https://developercommunity.visualstudio.com/t/python-package-cannot-be-installed-from-azure-arti/1434932?from=email&viewtype=all#T-ND1442840).

Can the size of the package description be reduced due to avoid this issue? It does seem to be quite a bit.

`
ERROR: Could not install requirement pydantic<3.0.0,>=2.4.2 from https://pkgs.dev.azure.com/ORGNAME/_packaging/FEED/pypi/download/pydantic/2.5/pydantic-2.5.0-py3-none-any.whl#sha256=7ce6e766c456ad026fe5712f7bcf036efc34bd5d107b3e669ef7ea01b3a9050c (from package-name==1.1.7) because of HTTP error 400 Client Error: Bad Request - The package description exceeds the maximum length of 324608 bytes. (DevOps Activity ID: F2025AE8-6C14-4F3A-8A30-C7439A994B54) for url: https://pkgs.dev.azure.com/ORGNAME/_packaging/FEED/pypi/download/pydantic/2.5/pydantic-2.5.0-py3-none-any.whl for URL https://pkgs.dev.azure.com/ORGNAME/_packaging/FEED/pypi/download/pydantic/2.5/pydantic-2.5.0-py3-none-any.whl#sha256=7ce6e766c456ad026fe5712f7bcf036efc34bd5d107b3e669ef7ea01b3a9050c (from https://pkgs.dev.azure.com/ORGNAME/_packaging/FEEDNAME/pypi/simple/pydantic/) (requires-python:>=3.7)
`

### Python, Pydantic & OS Version

```Text
pydantic version: 2.5.0 (this is after fiddling to get it to install, prior to installation the version is on V1, as that is what Databricks has installed by default)
        pydantic-core version: 2.14.1
          pydantic-core build: profile=release pgo=true
                 install path: /local_disk0/.ephemeral_nfs/envs/pythonEnv-ed43e11a-7019-4b48-a73f-836c180cf6da/lib/python3.10/site-packages/pydantic
               python version: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
                     platform: Linux-4.15.0-2101-aws-fips-x86_64-with-glibc2.35
             related packages: typing_extensions-4.8.0 typing_extensions-4.4.0 pyright-1.1.294 typing_extensions-4.7.1
```

**Comment by sergioumaya** [comment]
We are facing the same exact issue when trying to install pydantic 2.5.0 using azure devops artifact.

I support the solution proposed by @NodeJSmith  based on this [PR](https://github.com/Azure/azure-cli/pull/18485) in the azure-cli repo.

**Comment by sydney-runkle** [comment]
HI @NodeJSmith,

Thanks for reporting this. We're looking into a solution for this issue! 🐛

**Comment by sydney-runkle** [comment]
This has been fixed with the `2.5.1` release 👍
